engine/src/flutter/shell/platform/linux/fl_task_runner.h

```diff
@@ -25,17 +25,29 @@ G_DECLARE_FINAL_TYPE(FlTaskRunner, fl_task_runner, FL, TASK_RUNNER, GObject);
 FlTaskRunner* fl_task_runner_new(FlEngine* engine);
 
 /**
- * fl_task_runner_post_task:
+ * fl_task_runner_post_flutter_task:
  * @task_runner: an #FlTaskRunner.
  * @task: Flutter task being scheduled
  * @target_time_nanos: absolute time in nanoseconds
  *
  * Posts a Flutter task to be executed on main thread. This function is thread
  * safe and may be called from any thread.
  */
-void fl_task_runner_post_task(FlTaskRunner* task_runner,
-                              FlutterTask task,
-                              uint64_t target_time_nanos);
+void fl_task_runner_post_flutter_task(FlTaskRunner* task_runner,
+                                      FlutterTask task,
+                                      uint64_t target_time_nanos);
+
+/**
+ * fl_task_runner_post_callback:
+ * @task_runner: an #FlTaskRunner.
+ * @callback: callback to be scheduled
+ * @data: data to be passed to the callback
+ *
+ * Schedules arbitrary callback to be executed on main thread.
```

Comment:
Missing documentation on when it is executed - `fl_task_runner_post_flutter_task` has a scheduled time, when do callbacks fit in with that?